IFC4x3/Properties/s/SignalSightingAchievable_1NvUvG1d5BFRrtpV4UnXSI/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SignalSightingAchievable_1NvUvG1d5BFRrtpV4UnXSI" Name="SignalSightingAchievable" UniqueId="57e5ee50-0671-4b3d-bd77-cdf11ec61712" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Propose renaming to `SignalSightingDistance`.

Reply:
Updated to SignalSightingAchievableDistance: https://github.com/bSI-InfraRoom/IFC-Specification/pull/179/commits/b9fc777a565534c2a373dc537fcde287347974bb